controllers/admin.js

```diff
@@ -0,0 +1,122 @@
+import models from '../models';
+import hasher from '../helpers/hasher';
+
+const { users } = models;
+
+class Users {
+  static async createUser(req, res) {
+    const {
+      firstname, lastname, email, username, password
+    } = req.body;
+    try {
+      await users.create({
+        firstname,
+        lastname,
+        email,
+        username,
+        password: hasher(password), // Generating a hashed psaaword
+      });
+      return res.status(201).json({
+        message: 'user successfully created',
+        user: {
+          email,
+          username
+        }
+      });
+    } catch (error) {
+      if (error.original.code === '23505') {
+        return res.status(500).json({
```

Comment:
when a user is already reqistered, the status code to be returned should be `409` since there's a conflict not `500`